Docs: Sort log result documentation (#26898)
grafana/grafana: docs/sources/features/explore/index.md

Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.

#### Deduping

Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:

> - `exact` Exact matches are done on the whole line, except for date fields.

Reviewer: ```suggestion
- **exact -** Exact matches are done on the whole line except for date fields.
```

> - `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).

Reviewer: ```suggestion
- **numbers -** Matches on the line after stripping out numbers such as durations, IP addresses, and so on.
```

> - `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.

Reviewer: ```suggestion
- **signature -** The most aggressive deduping, this strips all letters and numbers and matches on the remaining whitespace and punctuation.
```

#### Flip results order

> You can change the order of received logs from the default descending order (newest first) to ascending order (oldest first).

Reviewer: This is a great change!
